src/logger/fileWriter.c

```diff
@@ -383,6 +390,9 @@ TESTABLE_STATIC int logging_sample(struct logging_status *ls,
                 if (WRITING_ACTIVE != ls->writing_status)
                         open_log_file(ls);
 
+                if (SD_CARD_NOT_PRESENT == ls->writing_status)
```

Comment:
Is this the correct location here since we try top open a log file above this?  I would double check this.